### PR TITLE
Mg/loading cloud array

### PR DESCRIPTION
**Pull request by MargrietGroenendijk**

### CODE REVIEW

File: src/widget.ts [hunk 1]
```diff
@@ -470,11 +470,18 @@ export class BabylonImageView extends BabylonBaseView {
...
+async function loadPointCloud(values: {name_space: string, array_name: string, bbox: { X: number[], Y: number[], Z: number[]}, token: string, tiledb_env: string}) {
 
-  const config = {
-    apiKey: values.token
-  };
+  const config: Record<string, any> = {};
+  
+  config.apiKey = values.token;
+
+  if(values.tiledb_env === "stage")
```
Comment (Shelnutt2) [inline src/widget.ts:479]: Can we make this configurable? Enterprise customers will need to set their own basePath.

File: pybabylonjs/args.py [hunk 2]
```diff
@@ -96,13 +97,26 @@ def check_point_cloud_data_local(mode, uri, kwargs):
     return data
 
 
-def check_point_cloud_data_cloud(kwargs):
+def check_point_cloud_data_cloud(uri, kwargs):
```
[the remark this answers is not in the document]
Comment (MargrietGroenendijk) [reply pybabylonjs/args.py:100]: updated

File: pybabylonjs/args.py [hunk 3]
```diff
@@ -96,13 +97,26 @@ def check_point_cloud_data_local(mode, uri, kwargs):
...
+        if token == None:
+            raise ValueError(
+                "The TileDB Cloud token needed to access the array is not specified or cannot be accessed"
+            )
+        kwargs = {**kwargs, "token": token}
+
     if not "bbox" in kwargs:
         raise ValueError("The bbox for slicing data from the array is not specified")
+
+    name_space = uri.split("://")[1].split("/")[0]
```
[the remark this answers is not in the document]
Comment (MargrietGroenendijk) [reply pybabylonjs/args.py:113]: updated

File: src/widget.ts (hunk 1 above)
Comment (MargrietGroenendijk) [reply src/widget.ts:479]: updated

**Pull request merged**